<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductWindow</class>
 <widget class="QMainWindow" name="ProductWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>264</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>406</width>
    <height>264</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>406</width>
    <height>264</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Poção Adquirida</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="productLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>71</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>35</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
